draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -205,14 +205,14 @@ data, and termination of sessions.
 DCCP, as stated in {{RFC4340}} does not provide reliable and ordered
 delivery. Consequently, multiple application subflows may be multiplexed over a
 single DCCP connection with no inherent performance penalty 
-for flows that do not require in-ordered delivery. DCCP
+for said subflows that do not require in-ordered delivery. DCCP
```

Comment:
Perhaps "application subflows" instead?

Reply:
i wanted to avoid repetition. But i changed it to application sub-flows.